packtools/sps/models/funding_group.py

```diff
@@ -50,4 +50,36 @@ def funding_sources(self):
 
     @property
     def funding_statement(self):
-        return self._xmltree.xpath(".//funding-group/funding-statement")[0].text
+        funding_statements = self._xmltree.xpath(".//funding-group/funding-statement")
+        if funding_statements:
+            return funding_statements[0].text
```

Comment:
@Rossi-Luciano se é sabido que é um item, pode usar `self._xmltree.findtext(".//funding-group/funding-statement")`

Reply:
No entanto, é importante verificar se o elemento `funding-statement` contém ou não outros elementos, pois text somente retornará o texto entre o início do elemento até o próximo elemento, por exemplo:
```xml
<root><elemento1>texto 1<b>bold</b> texto 2</elemento1></root>
```
elemento1.text retornará apenas `texto 1`
` texto 2` é b.tail

Reply:
OK @robertatakenaka